mappings/net/minecraft/reference/Sounds.mapping

```diff
@@ -7,24 +7,24 @@ CLASS none/nl net/minecraft/reference/Sounds
 	FIELD F ENTITY_BLAZE_SHOOT Lnone/nk;
 	FIELD G ENTITY_BOBBER_SPLASH Lnone/nk;
 	FIELD H ENTITY_BOBBER_THROW Lnone/nk;
-	FIELD I ITEM_BOTTLE_FILL Lnone/nk;
```

Comment:
Just grab the old Sounds.mapping (revert) and don't touch this manually. Text editors are better for these kinds of mass movements.